libs/wlgen/wlgen/rta.py

```diff
@@ -98,7 +98,7 @@ def calibrate(target):
         # Setup logging
         log = logging.getLogger('RTApp')
 
-        # target.cpufreq.save_governors()
+        old_governors = target.cpufreq.get_all_governors()
```

Comment:
I disagree! There's no shortage of characters here and `old_governors` is significantly more accurate IMO.